intro-HPC/ch_troubleshooting.tex

```diff
@@ -20,14 +20,93 @@ \section{Walltime issues}
 \section{Out of quota issues}
 
 Sometimes a job hangs at some point or it stops writing in the disk. These errors are usually
-related to the quota usage. You may have reached your quota limit at some storage endpoint. 
+related to the quota usage. You may have reached your quota limit at some storage endpoint.
 You should move (or remove) the data to a different storage endpoint (or request more quota) to be able to write to the disk and then resubmit the jobs.
 \ifgent
 Another option is to request extra quota for your VO to the VO moderator/s.
-See section~\ref{subsec:predefined-user-directories} and section~\ref{subsec:predfined-quotas} for more information about 
+See section~\ref{subsec:predefined-user-directories} and section~\ref{subsec:predfined-quotas} for more information about
 quotas and how to use the storage endpoints in an efficient way.
 \fi
 % FIXME: Add how to request quota section
 
+\section{Issues connecting to login node}
+\label{sec:connecting-issues}
 
+If you have errors that look like
 
+\begin{prompt}
+%\userid{}%@%\loginnode%: Permission denied
+\end{prompt}
+
+
+
+If you have issues connecting to the login nodes, here is a list of things to do
+that might help:
+
+\begin{enumerate}
+    \item It can take up to an hour for you VSC account to become active after
+        it has been \emph{approved}.
+\ifmacORlinux
+    \item Your SSH private key may not be in the default location (\verb|$HOME/.ssh/id_rsa|).
+        There are several ways to deal with this (using one of these is sufficient):
+\begin{enumerate}
+            \item Use the \verb|ssh -i| (see section~\ref{sec:connect}) \emph{OR;}
+            \item Use \verb|ssh-add|:
+                \iflinux
+                see section~\ref{sec:using-ssh-agent-with-openssh}
+                \else
+                see this \href{https://apple.stackexchange.com/a/250572}{answer on StackExchange}.
```

Comment:
Still not happy with this... But if you insist on including it, it should be in the SSH agent section, not here?